runtime/vara/Cargo.toml

```diff
@@ -177,7 +184,6 @@ std = [
 	"sp-version/std",
 	"substrate-wasm-builder",
 	"pallet-airdrop/std",
-	"validator-set/std",
 ]
 runtime-benchmarks = [
 	"frame-benchmarking/runtime-benchmarks",
```

Comment:
Don't we need to append here `runtime-benchmarks` feature for added pallets, bench weights for them and use in the runtime?